influxdb_client/client/influxdb_client.py

```diff
@@ -93,6 +93,9 @@ def from_config_file(cls, config_file: str = "config.ini", debug=None, enable_gz
             - timeout,
             - verify_ssl
             - ssl_ca_cert
+            - cert_file
+            - key_file
+            - key_password
```

Comment:
Please rename options:

```suggestion
            - cert_file
            - cert_key_file
            - cert_key_password
```